frontend/src/metabase/admin/settings/notifications/WebhookForm.tsx

```diff
@@ -0,0 +1,212 @@
+    throw e;
+  }
+};
+
+export const WebhookForm = ({
+  onSubmit,
+  onCancel,
+  onDelete,
+  initialValues,
+  submitLabel = t`Create destination`,
+}: {
+  onSubmit: (props: WebhookFormProps) => void;
+  onCancel: () => void;
+  onDelete?: () => void;
+  initialValues: WebhookFormProps;
+  submitLabel?: string;
+}) => {
+  const { label: testButtonLabel, setLabel: setTestButtonLabel } =
+    useActionButtonLabel({ defaultLabel: t`Send a test` });
+  const [testChannel] = useTestChannelMutation();
+
+  const handleTest = async (
+    values: WebhookFormProps,
+    setFieldError: WebhookFormikHelpers["setFieldError"],
+  ) => {
+    await testChannel({
+      details: {
+        url: values.url,
+        "auth-method": values["auth-method"],
+        "auth-info": values["auth-info"],
+      },
+    })
+      .unwrap()
+      .then(
+        () => {
+          setFieldError("url", undefined);
+          setTestButtonLabel(t`Success`);
+        },
+        e => {
+          setTestButtonLabel(t`Test failed`);
+          if (typeof e === "string") {
+            setFieldError("url", e);
+          } else if (typeof e.data === "string") {
+            setFieldError("url", e.data);
+          } else if (e.data?.message) {
+            setFieldError("url", e.data.message);
+          }
+        },
+      );
+  };
+
+  return (
+    <FormProvider
+      initialValues={initialValues}
+      onSubmit={onSubmit}
+      validationSchema={validationSchema}
+    >
+      {({ dirty, values, setFieldError }) => (
+        <Form>
+          <Alert
+            variant="light"
+            mb="1.5rem"
+            style={{ backgroundColor: "var(--mb-color-bg-light)" }}
+            px="1.5rem"
+            py="1rem"
+            radius="0.5rem"
+          >
+            <Text>{t`You can send the payload of any Alert to this destination whenever the Alert is triggered. Learn about Alerts`}</Text>
+          </Alert>
+          <Flex align="end" mb="1.5rem" gap="1rem">
+            <FormTextInput
+              name="url"
+              label="Webhook URL"
+              placeholder="http://hooks.example.com/hooks/catch/"
+              styles={styles}
+              style={{ flexGrow: 1 }}
+              maw="21rem"
+            />
+            <Button
+              h="2.5rem"
+              onClick={() => handleTest(values, setFieldError)}
+            >
+              {testButtonLabel}
+            </Button>
+          </Flex>
+          <FormTextInput
+            name="name"
+            label="Give it a name"
+            placeholder="Hooky McHookerson"
+            styles={styles}
+            mb="1.5rem"
+            maw="14.5rem"
+          />
+          <FormTextInput
+            name="description"
+            label="Description"
+            placeholder="Where is this going and what does it send?"
+            styles={styles}
+            mb="1.5rem"
+            maw="21rem"
+          />
+          <FormChipGroup
```

Comment:
nit: The four chips, side by side, look really good! There's a generous amount of vertical spacing in the group, so in certain cases (depending on locale or viewport width) the last item may wrap like this:

![image](https://github.com/user-attachments/assets/f9f0f2ad-864d-43e5-b178-1302d9582cf6)

This may be a tiny bit confusing. There seems to be a lot of padding on each chip. Maybe that could be removed, and then the group could apply a consistent amount of spacing (horizontally and vertically) to all the chips, so they look good even when they wrap